gensim/matutils.py

```diff
@@ -1424,183 +1424,6 @@ def close(self):
 
 try:
     # try to load fast, cythonized code if possible
-    from gensim.corpora._mmreader import MmReader
+    from gensim.corpora._mmreader import MmReader  # noqa
```

Comment:
why? this must be always available, no more try/exc wth `FAST_VERSION`/`NO_CYTHON`/etc